email/te-submission-email.html

```diff
@@ -0,0 +1,56 @@
+<body>
+  <p>TE form submitted.</p>
+
+  <div style="color:red; font-weight:bold;">
```

Comment:
All this block should be replaced with the warnings passed from the function as a string array. See `sqr-submission` template for example.